src/afl-fuzz-init.c

```diff
@@ -2915,11 +2915,16 @@ void setup_testcase_shmem(afl_state_t *afl) {
   afl->shm_fuzz = ck_alloc(sizeof(sharedmem_t));
 
   // we need to set the non-instrumented mode to not overwrite the SHM_ENV_VAR
-  u8 *map = afl_shm_init(afl->shm_fuzz, MAX_FILE + sizeof(u32), 1);
+  size_t shm_fuzz_page_size = MAX_FILE + sizeof(u32);
+  u8 *map = afl_shm_init(afl->shm_fuzz, shm_fuzz_page_size, 1);
```

Comment:
`afl_shm_init` is used at multiple places, if I remember it correctly. I think it would be better to overwrite the map size inside `afl_shm_init` for consistency?

Reply:
Oh so this is the initial map size and we take the sizes from forkserver afterwards?

Reply:
Oh, wait, that's coverage map size. Silly me =p.